tests/unit/test_business_logic.py

```diff
@@ -436,15 +446,6 @@ def test_request_all_files_set(bll):
     assert len(filegroup.supporting_files) == 1
 
 
-def test_request_release_invalid_state():
```

Comment:
Removed because mypy catches this error now

Reply:
woo